moto/glue/responses.py

```diff
@@ -208,6 +208,35 @@ def update_partition(self):
 
         return ""
 
+    def batch_update_partition(self):
+        database_name = self.parameters.get("DatabaseName")
+        table_name = self.parameters.get("TableName")
+        table = self.glue_backend.get_table(database_name, table_name)
+
+        errors_output = []
+        for entry in self.parameters.get("Entries"):
+            part_to_update = entry["PartitionValueList"]
+            part_input = entry["PartitionInput"]
+
+            try:
+                table.update_partition(part_to_update, part_input)
+            except PartitionNotFoundException:
+                errors_output.append(
```

Comment:
Hi @dkuntz2, thanks for the PR!
Can we get a unit test for this branch as well? Would be good to assert that some batch-entries are created, but errors are returned for failing batch-entries.

Reply:
Test case added